src/lib.rs

```diff
@@ -247,6 +270,13 @@ pub fn metadata_deps(manifest_path: Option<&Path>, deps: bool) -> Result<Metadat
         cmd.arg("--no-deps");
     }
 
+    match feat {
+        CargoOpt::AllFeatures => cmd.arg("--features-all"),
+        CargoOpt::NoDefaultFeatures => cmd.arg("--no-default-features"),
+        CargoOpt::SomeFeatures(ftrs) => cmd.arg(format!("--fatures {:?}", ftrs)),
+        CargoOpt::DefaultFeatures => cmd.arg(""),
```

Comment:
This arm should be a nop `{}` arm instead of adding an empty argument

Reply:
is not possible to give a nop arg because cmd.arg returns a Command. Suggestions?

Reply:
Just make the last arm `cmd` instead of `cmd.arg("")`, that should do it.